Add dynamic generation of `robots.txt` (#6692)
withastro/docs: src/content/docs/en/guides/integrations-guide/sitemap.mdx

Sitemap: https://<YOUR SITE>/sitemap-index.xml
```

> If you'd like to generate `robots.txt` dynamically, add a file named `robots.txt.ts`.

Reviewer: ```suggestion
To generate `robots.txt` dynamically, add a file named `robots.txt.ts` with the following code:
```
We can make this even shorter and more direct!

> If you'd like to generate `robots.txt` dynamically, add a file named `robots.txt.ts`. In the following code, we're using the [`dedent`](https://www.npmjs.com/package/dedent) package for formatting multi-line template literals.

Reviewer 2: We do not recommend using the first person in documents.

You can find writing and style Guide in [Astro Docs Docs](https://contribute.docs.astro.build/guides/writing-style/#writing-style)
